web/modules/custom/ingredient_term/ingredient_term.module

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * @file
+ * Contains hook for Ingredient Term module.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\taxonomy\Entity\Term;
+
+/**
+ * Implements hook_entity_presave().
+ */
+function ingredient_term_entity_presave(EntityInterface $entity): void {
```

Comment:
Use the `hook_ENTITY_TYPE_presave` hook for node entity presaves. If `node` is specified you should use `NodeInterface` to typehint the $entity object.